src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx

```diff
@@ -601,12 +601,28 @@ const MeasureGroups = (props: MeasureGroupProps) => {
         .expressionDefinitions;
       setExpressionDefinitions(definitions);
     }
-    if (measure && (measure.cqlErrors || !measure?.cql)) {
+    if (measure && (measure.cqlErrors || !measure?.cql) && !measure?.scoring) {
+      // bad cql and bad base config step
+      setAlertMessage(() => ({
+        type: "error",
+        message:
+          "Please complete the CQL Editor process and Base Configuration tab before continuing",
+        canClose: false,
+      }));
+    } else if (measure && (measure.cqlErrors || !measure?.cql)) {
```

Comment:
thinking...could this be the first condition in the `if...else if` chain? this way I think we can completely get rid of the first `if`. Also, we are checking the `if measure &&` so do not think we need to use safe navigation here `!measure?.cql` again.

Reply:
I think we still need the first one in that case since cql can be null